docs/source/fundamentals/benefits.md

```diff
@@ -0,0 +1,133 @@
+    }}
+  </Query>
+);
+```
+
+Apollo Client takes care of the request cycle from start to finish, including tracking loading and error states for you. There’s no middleware to set up or boilerplate to write before making your first request, nor do you need to worry about transforming and caching the response. All you have to do is describe the data your component needs and let Apollo Client do the heavy lifting. 💪
+
+You’ll find that when you switch to Apollo Client, you’ll be able to delete a lot of unnecessary code related to data management. The exact amount will vary depending on your application, but some teams have reported up to thousands of lines. To learn more about how Apollo Client enables advanced features like optimistic UI, refetching, and pagination with less code, check out our guide on [state management](../guides/state-mgmt.html).
+
+<h2 id="performance">Improved performance</h2>
+
+In many cases, layering a GraphQL API over your existing REST endpoints can improve your app's performance, especially on devices with slow network connections. While you should always measure to determine how integrating GraphQL will affect your application, it's generally accepted that GraphQL improves performance by helping avoid round trips to the server and reducing payload size.
+
+<h3 id="payload">Smaller payloads</h3>
+
+Since the response back from the server contains only the properties you specify in your query, GraphQL can significantly reduce payload size compared to a REST endpoint. Let's take a look at our dogs query from earlier in the article:
+
+```graphql
+const GET_DOGS = gql`
+  query {
+    dogs {
+      id
+      breed
+      image {
+        url
+      }
+      activities {
+        name
+      }
+    }
+  }
+`;
+```
+
+The response back from the server will be a list of dog objects with `id`, `breed`, `image`, and `activities` properties. It doesn't matter if the underlying REST endpoints we call in our resolvers return back objects with 100 properties! All of those extraneous properties will be filtered out before the response is sent back to the client.
+
+<h3 id="round-trip">Avoid round trips</h3>
+
+Since each GraphQL request returns only one response, switching to GraphQL can help you avoid costly round trips from the client to your server. With REST, each resource represents a round trip, which can quickly add up. If you're fetching items in a list, you'll have to complete a round trip for every resource multiplied by the number of items, causing slow load times especially on mobile devices.
+
+```bash
+GET /api/dogs/breeds
+GET /api/dogs/images
+GET /api/dogs/activities
+```
+
+With GraphQL, each query represents a single round trip from the client to server. If you'd like to reduce round trips even further, you can implement [query batching](https://www.apollographql.com/docs/engine/query-batching.html) to batch multiple queries into a single request. 
+
+<h2 id="dev-experience">Developer experience</h2>
```

Comment:
I love this whole section `Developer experience` section and I feel like it should be higher in this portion of the guide!  Can we put it first?

Reply:
I'm cool with that 👍